scripts/webpack/webpack.swagger.js

```diff
@@ -0,0 +1,26 @@
+// @ts-check
+const path = require('path');
+
+const makeBaseConfig = require('./webpack.prod.js');
+
+module.exports = (env = {}) => {
+  const baseConfig = makeBaseConfig(env);
+
+  return {
+    ...baseConfig,
+
+    entry: {
+      app: './public/swagger/index.tsx',
```

Comment:
^^ @joshhunt I changed the name to match core grafana -- then the backend can look for the same properties